src/pages/rooms/room-creation/room-creation.page.jsx

```diff
@@ -1,9 +1,180 @@
 import "./room-creation.page.scss";
+import { Endpoints, States } from "../../../constants";
+import { useState, useEffect } from "react";
+import {
+  Box,
+  TextField,
+  Button,
+  FormControl,
+  FormLabel,
+  RadioGroup,
+  FormControlLabel,
+  Radio,
+  InputLabel,
+  Select,
+  MenuItem,
+} from "@mui/material";
+import SendIcon from "@mui/icons-material/Send";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import { useNavigate } from "react-router-dom";
+import AuthenticationService from "../../../services/authentication.service";
+import * as signalR from "@microsoft/signalr";
 
 const RoomCreationPage = () => {
-    return (
-        <div className="qcg-room-creation-page"></div>
-    );
+  const authenticationService = new AuthenticationService();
+
+  const [roomName, setRoomName] = useState();
+  const [category, setCategory] = useState(" ");
```

Comment:
useState should start with an empty  string with no spaces useState(""). Currently, it's creating a warning for the MUI (check the console).